syft/__init__.py

```diff
@@ -1 +1,2 @@
 from .tensor import TensorBase
+from .tensor import *
```

Comment:
You can either do `from .tensor import x, y, z`, or if the number of methods is large, you can just do `import .tensor` and then do use `tensor.x` in code.